docs/Project.toml

```diff
@@ -2,6 +2,8 @@
 ArtifactWrappers = "a14bc488-3040-4b00-9dc1-f6467924858a"
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
+ClimaLSM = "7884a58f-fab6-4fd0-82bb-ecfedb2d8430"
```

Comment:
do we need to also check in the Manifest for docs now? Because we need the dev version of ClimaLSM, not the latest registered version?

Reply:
Hmm, I agree that if ClimaLSM is in the Project, we should check in the Manifest. But I'm confused why ClimaLSM isn't already in this Project before this PR. On main, there are files in docs/ that use ClimaLSM (e.g. [bucket_tutorial](https://github.com/CliMA/ClimaLSM.jl/blob/main/docs/tutorials/Bucket/bucket_tutorial.jl#L140))

Reply:
I think that it is a mix of using the registered version and things like this:
https://github.com/CliMA/ClimaLSM.jl/blob/c9a04b631b7ad01c73609a533286e30ff787d8d9/docs/tutorials/Soil/richards_equation.jl#L64

the LOADPATH command lets you use the current version, because when we build the docs, we build from ClimaLSM.jl. We used to do this for test and for experiments too. But it was confusing.

Reply:
/is confusing :)

Reply:
Thank you!